### ISSUE TITLE
[Question] Does OpenVR support the Oculus Go?

### ISSUE THREAD

**Issue by cyrilzakka** [ValveSoftware/openvr#990]
I can't seem to find a list of supported devices anywhere on this GitHub page or SteamVR. Does OpenVR support the Oculus Go or only the Rift?

**Comment by BOLL7708** [comment]
Both the Go and Mirage Solo are standalone ARM systems right? I don't think Steam nor SteamVR exist for ARM currently. We only have a companion app for ARM so far.

Actually, to my knowledge there doesn't exist any standalone unit at all that runs SteamVR/OpenVR, not commercially at least, unless we count backtops but I don't as those are separate from the headset. 

At least the Vive Cosmos is straddling the desktop/mobile VR gulf being able to connect to either a SteamVR/OpenVR desktop or supposedly some kind of mobile device, details are still a bit fuzzy.

**Comment by cyrilzakka** [comment]
Oh alright thank you for the prompt reply. I'm looking to control/interact with a VR environment running on my Mac (a MuJoCo environment actually) using a VR headset but the only ones I have available to me are the Go and Mirage so standalone headsets. Would you know of any alternatives?

**Comment by TheDeveloperGuy** [comment]
Unless I'm mistaken, the only VR headset with an OpenVR macOS driver is the Vive/Vive Pro.

**Comment by cyrilzakka** [comment]
@TheDeveloperGuy yeah after looking around that's what I've figured. I'm thinking of porting my environment inside of Unity and loading it onto on the headset itself while in Development mode. Seems like the best way to do it. Just hoping I don't hit any more roadblocks.

**Comment by sholtodouglas** [comment]
@cyrilzakka did you get this to work? I'm interested in doing the same, was hoping to run pybullet sims and teleoperate through VR
